### PR TITLE
Add maintainer listing scripts

### PR DESCRIPTION
**Pull request by jgrassler**
This commit adds a collection of scripts for generating a list
of maintainers for packages shipped as part of SUSE OpenStack
Cloud 7, 8 and 9.

### CODE REVIEW

**Review by JanZerebecki — APPROVED** [review]
Not reviewed or tested.

**Review by JanZerebecki — DISMISSED** [review]
Please appease bashhate and flake8 as seen in travis.

File: scripts/maintainers/cloudpackages.py [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+            return _PROJECT_REPO[key]
+    return None
+
+
+def find_groupfiles(product_file):
+    """
+    Extract a list of all group files being included from a product file.
+    """
+    group_files = []
+    f = open(product_file)
```
Comment (skazi0) [inline scripts/maintainers/cloudpackages.py:224]: why not `with` ?

File: scripts/maintainers/genmaintainers.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    echo "  cloud_release:: 7 | 8 | 9"
+    }
+
+if [ -z $cloud_release ]; then
+    usage
+    exit 1
+fi
+
+case $cloud_release in
+    "7")
```
Comment (skazi0) [inline scripts/maintainers/genmaintainers.sh:33]: Maybe:
```
case $cloud_release in
	7|8|9)
		;;
	*)
		usage
		exit 1
esac

**Review by skazi0 — APPROVED** [review]
looks good!

**Review by JanZerebecki — APPROVED** [review]

**Pull request merged**